cmd/nvidia-ctk/cdi/generate/generate.go

```diff
@@ -221,7 +231,7 @@ func (m command) validateFlags(c *cli.Context, opts *options) error {
 	return nil
 }
 
-func (m command) run(c *cli.Context, opts *options) error {
+func (m command) run(_ *cli.Context, opts *options) error {
```

Comment:
I suppose we COULD use it, but we don't really make use of non-`cli.Contexts` in this particular codebase.

Since this change is actually unrelated to anything else we're doing here, I'll rather roll it back. I'm happy to revisit in future.